python/nx-cugraph/nx_cugraph/classes/graph.py

```diff
@@ -649,6 +666,7 @@ def _get_plc_graph(
             store_transposed=store_transposed,
             renumber=False,
             do_expensive_check=False,
+            vertices_array=self._node_ids,
```

Comment:
Cool! I didn't realized `vertices_array` made it in. Also, the recent `drop_self_loops` and `drop_multi_edges` may be useful too 👍